updatefiles-tofix-rebooting-container

ibrarakram opened the pull request:
The container is automatically power cycling and this is to do with the package.jason / YMAL file configuration.

paul-mitchell-bjss commented on server.js:
```diff
@@ -133,7 +133,8 @@ utils.findAvailablePort(app, function (port) {
         notify: false,
-        logLevel: 'error'
+        logLevel: 'error',
+        reloadDelay: 1000
```
Not really - it's more that the styles take a while to compile and without this delay I was getting poor page loads

paul-mitchell-bjss replied:
Which may be due to the change I made to the gulp watch tasks - I'll check

paul-mitchell-bjss replied:
If I switch back to using runSequence in the gulp.watch task and take out the reload delay, I still get bad page reloads. Looks like adding in the scripts just tipped the timing balance on reloads, so the delay is necessary now.

visualunderground replied:
Fair enough

visualunderground commented on app/views/index.hbs:
```diff
@@ -5,4 +5,5 @@
 {{#if showDocumentation}}
 <p><a href="/docs/guides/index.md">Documentation</a></p>
 <p><a href="/components">UI Library</a></p>
+<p><a href="/examples/index">Examples</a></p>
```
Examples -> Example Prototypes

paul-mitchell-bjss replied:
Agreed

visualunderground commented on app/views/examples/index.hbs:
```diff
@@ -0,0 +1,7 @@
+<h1>NHSUK Prototyping Kit</h1>
+<h2>Example Prototypes</h2>
+<ul>
+  <li><a href="/examples/test">Smoke Test</a>
```
What's a "Smoke test", is there a better name?

paul-mitchell-bjss replied:
Well, it's an example intended to exercise the basic operation of the kit. I did just call it "Test" before. Perhaps "Kit Test Prototype"?

visualunderground replied:
Is it a "Simple example", "Basic use", or "Passing data to templates"?

paul-mitchell-bjss replied:
Passing Data to Templates - genius!

paul-mitchell-bjss replied:
I've gone with "Test passing data to a template"

ibrarakram committed:
updatefiles-tofix-rebooting-container

The container is automatically power cycling and this is to do with the package.jason / YMAL file configuration.

ibrarakram committed:
Update azure-pipelines.yml

ibrarakram committed:
Update azure-pipelines.yml
